core/primitives/src/epoch_info.rs

```diff
@@ -480,6 +480,27 @@ impl EpochInfo {
         }
     }
 
+    pub fn get_validator_stake(&self, account_id: &AccountId) -> Option<Balance> {
```

Comment:
nit: you can call `get_validator_by_account(account_id).map(|vs| vs.stake)` as well.

Reply:
I decided to add this to avoid cloning the entire `ValidatorStake`.